docs/source/dev_guide/release.rst

```diff
@@ -3,124 +3,85 @@ How to Prepare a Release
-and draft a new release.
-
-2. ``Tag version`` and ``Release title`` should both be the version, including the "v".
-(They should match ``git_rev`` in step 4 of the previous section).
-``Target`` should be ``main``. Use ``Describe this release`` to write what features
-the release adds. You can scroll through
-`zppy commits <https://github.com/E3SM-Project/zppy/commits/main>`_ to see
-what features have been added recently.
-
-Note that you can also change the branch which you want to release from,
-this is specified after the tag (@ Target: ``main``).
-
-The title of a release is often the same as the tag, but you can set it to whatever you want.
+Releasing on GitHub
+-------------------
 
-Remember to write a description.
+1. Draft a new release `here <https://github.com/E3SM-Project/e3sm_diags/releases>`_.
+2. Set `Tag version` to ``v<version>``, **including the "v"**. `@Target` should be ``master``.
+3. Set `Release title` to ``v<version>``, **including the "v"**.
+4. Use `Describe this release` to summarize the changelog.
 
-3. Click "Publish release".
+   * You can scroll through `zppy commits <https://github.com/E3SM-Project/zppy/commits/main>`_ for a list of changes.
 
-Releasing The Software On Anaconda
-----------------------------------
+5. If this version is a release candidate (``<version>`` appended with ``rc``), checkmark `This is a pre-release`.
+6. Click `Publish release`.
+7. CI/CD release workflow is automatically triggered.
 
-1. Be sure to have already completed :ref:`Creating A Release On GitHub <github-release>`.
-This triggers the CI/CD workflow that handles Anaconda releases.
+Releasing Anaconda
+------------------
 
+1. Be sure to have already completed :ref:`Creating A Release On GitHub <github-release>`. This triggers the CI/CD workflow that handles Anaconda releases.
 2. Wait until the CI/CD build is successful. You can view all workflows at `All Workflows <https://github.com/E3SM-Project/zppy/actions>`_.
-
 3. Check the https://anaconda.org/e3sm/zppy page to view the newly updated package.
 
-4. Notify the maintainers of the unified E3SM environment about the new release on the
-`E3SM Confluence site <https://acme-climate.atlassian.net/wiki/spaces/WORKFLOW/pages/129732419/E3SM+Unified+Anaconda+Environment>`_.
-Be sure to only update the ``zppy`` version number in the correct version(s) of
-the E3SM Unified environment. This is almost certainly one of the versions listed under
-“Next versions”. If you are uncertain of which to update, leave a comment on the page
-asking.
+   * Release candidates are assigned the ``e3sm_dev`` label
+   * Production releases are assigned the ``main`` label
 
-Creating a New Version of the Documentation
--------------------------------------------
+4. Notify the maintainers of the unified E3SM environment about the new release on the `E3SM Confluence site <https://acme-climate.atlassian.net/wiki/spaces/WORKFLOW/pages/129732419/E3SM+Unified+Anaconda+Environment>`_.
```

Comment:
You might notice that long bullets are now single line instead of multiple lines.
By sacrificing line length in the `.rst` files, it allows bullet points to be compact and not require line breaks.